guides/image-sources/provisioning-installer-bootdisk-fullhost.plantuml

```diff
@@ -0,0 +1,63 @@
+@startuml
+
+!include foreman.pstyle
+!$networkboot = 0
+
+title Installer-based provisioning with full-host boot disk on bare metal
+
+actor       User
+participant "Provisioned\nHost" as Host
+participant "Foreman" as Foreman
+participant "Foreman\nProxy" as Proxy
+participant DNS
+!if ($puppet)
+participant "Puppet\nserver" as Puppet
+!endif
+
+note over Host : Powered off
+
+== Create host in Foreman ==
+
+User -> Foreman : Create host
+User -> Foreman : Assign a static IP address
+group Templates [*PXE* installer boot]
+    Foreman -> Proxy : Render boot-loader configuration files
+end
+Proxy -> Proxy : Deploy bootloader configuration files\nto ""/var/lib/tftpboot""
+Foreman -> Proxy : Command to download installer kernel\nand initial RAM disk
+Proxy -> Proxy : Download kernel and initial RAM disk
+Foreman -> Proxy : Create DNS records
+Proxy -> DNS : Forward DNS records
+Proxy -> Foreman : Report status of TFTP, DHCP & DNS
+Foreman -> User : Host is created\n(build mode enabled)
+
+User -> Foreman : Download the boot-disk ISO of the host
+User -> User : Write the boot-disk ISO\nto an external storage device
+User -> Host : Configure to boot\nfrom the external storage device
+
+== Boot into OS installer ==
+
+User -> Host : Power on
+Host -> Host : Use the IP address from the boot disk
+Host -> Host : Boot from the external storage device
+Host -> Host : Load operating system installer
+User -> Host : Eliminate the external storage device\n(too soon?)
```

Comment:
TBH, I don't know when it's happening, but is it necessary to mention it?

Reply:
We agreed that Leos will investigate when and how this happens.

Reply:
Updated.